Makefile

```diff
@@ -1,10 +1,25 @@
 PACKAGE_NAME = image-builder
 
+.PHONY: help
+help:
+	@echo "make [TARGETS...]"
+	@echo
+	@echo "This is the maintenance makefile of image-builder. The following"
+	@echo "targets are available:"
+	@echo
+	@echo "    help:               Print this usage information."
+	@echo "    build:              Build the project from source code"
+	@echo "    test:               Run tests"
+	@echo "    run:                Run the project on localhost"
+
 .PHONY: build
 build:
 	go build -o image-builder ./cmd/image-builder/
 	go build -o gen-oscap ./cmd/oscap
 	go build -o image-builder-migrate-db-tern ./cmd/image-builder-migrate-db-tern/
+
+.PHONY: test
+test:
 	go test -c -tags=integration -o image-builder-db-test ./cmd/image-builder-db-test/
```

Comment:
So this just compiles the tests, you'd still have to run the tests manually.

Reply:
Agreed, that makes it misleading. @elkoniu Could you please fix that up? Thanks! :heart:

Reply:
Thank you for the review, I have just pushed updated version.

Reply:
Since the `test` target is dropped, this is also resolved.